lib/git_reflow/git_server/git_hub/pull_request.rb

```diff
@@ -0,0 +1,108 @@
+require 'git_reflow/git_server/pull_request'
+
+module GitReflow
+  module GitServer
+    class GitHub
+      class PullRequest < GitReflow::GitServer::PullRequest
+        def initialize(attributes)
+          self.number              = attributes.number
+          self.description         = attributes[:body]
+          self.html_url            = attributes.html_url
+          self.feature_branch_name = attributes.head.label
+          self.base_branch_name    = attributes.base.label
+          self.source_object       = attributes
+          self.build_status        = build.state
+        end
+
+        def self.create(options = {})
+          self.new(GitReflow.git_server.connection.pull_requests.create(
+            GitReflow.git_server.class.remote_user,
+            GitReflow.git_server.class.remote_repo_name,
+            title: options[:title],
+            body:  options[:body],
+            head:  "#{GitReflow.git_server.class.remote_user}:#{GitReflow.git_server.class.current_branch}",
+            base:  options[:base]))
+        end
+
+        def self.find_open(to: 'master', from: GitReflow.git_server.class.current_branch)
+          matching_pull = GitReflow.git_server.connection.pull_requests.all(GitReflow.git_server.class.remote_user, GitReflow.git_server.class.remote_repo_name, base: to, head: "#{GitReflow.git_server.class.remote_user}:#{from}", state: 'open').first
+          if matching_pull
+            self.new matching_pull
+          end
+        end
+
+        # override attr_reader for auto-updates
+        def build_status
+          @build_status ||= build.state
+        end
+
+        def commit_author
+          begin
+            username, branch = base.label.split(':')
+            first_commit = GitReflow.git_server.connection.pull_requests.commits(username, GitReflow.git_server.class.remote_repo_name, number.to_s).first
+            "#{first_commit.commit.author.name} <#{first_commit.commit.author.email}>".strip
+          rescue Github::Error::NotFound
+            nil
+          end
+        end
+
+        def reviewers
+          comment_authors
+        end
+
+        def approvals
+          pull_last_committed_at = get_commited_time(self.head.sha)
```

Comment:
This is picky, but you have `committed` (correct) and `commited` (wrong spelling) used here